cds-snc/c19-benefits-node
# Small Text Changes

# Description
Closes: https://trello.com/c/BI40hjEt/185-small-text-changes
|Page|Figma|Site|
|---|---|---|
|M2a|![image](https://user-images.githubusercontent.com/87738/83834322-08cdf800-a6bc-11ea-8770-ad54b049c793.png)|![image](https://user-images.githubusercontent.com/87738/83834292-f5bb2800-a6bb-11ea-8b44-f8e5b3275374.png)|
|M6|![image](https://user-images.githubusercontent.com/87738/83834371-2ac77a80-a6bc-11ea-9792-d71b01992dbf.png)|![image](https://user-images.githubusercontent.com/87738/83834504-72e69d00-a6bc-11ea-8ecf-cb805e5d47d0.png)|
|mResults1|![image](https://user-images.githubusercontent.com/87738/83834694-ed172180-a6bc-11ea-9937-96392bedba3a.png)|![image](https://user-images.githubusercontent.com/87738/83834713-facca700-a6bc-11ea-8a60-8077c74fc420.png)|
|M10|![image](https://user-images.githubusercontent.com/87738/83834967-92ca9080-a6bd-11ea-8b82-3f0a9f38baed.png)|![image](https://user-images.githubusercontent.com/87738/83834873-5dbe3e00-a6bd-11ea-950a-18f5e432c04f.png)|
|M10 French|![image](https://user-images.githubusercontent.com/87738/83834985-a118ac80-a6bd-11ea-9d41-ffa6c69eb783.png)|![image](https://user-images.githubusercontent.com/87738/83834916-7595c200-a6bd-11ea-88f9-f2147dba2c36.png)|

## Test Instructions

1. Navigate to the following pages:
 M2: Select *Vous avez perdu votre revenu en entier.* when prompted.
 M6: Select *Non* or *Vous n’êtes pas certain.* on *Allocation canadienne pour enfants*
 MResults1: Ensure you have 1 benefit.
M10: Navigate through the site you'll get there eventuall.

## Help Requested

- Validate correct text.
